src/main/kotlin/apply/application/ApplicantService.kt

```diff
@@ -16,22 +16,33 @@ import javax.annotation.PostConstruct
 @Transactional
 @Service
 class ApplicantService(
+    private val applicationFormService: ApplicationFormService,
     private val applicantRepository: ApplicantRepository,
     private val cheaterRepository: CheaterRepository,
     private val jwtTokenProvider: JwtTokenProvider
 ) {
-    fun findAll(): List<ApplicantResponse> = applicantRepository.findAll().map {
-        ApplicantResponse(it, cheaterRepository.existsByApplicantId(it.id))
+    fun findAll(): List<ApplicantBasicResponse> = applicantRepository.findAll().map {
+        ApplicantBasicResponse(it)
     }
 
-    fun findAllByIds(applicantIds: List<Long>): List<ApplicantResponse> =
-        applicantRepository.findAllById(applicantIds).map {
-            ApplicantResponse(it, cheaterRepository.existsByApplicantId(it.id))
+    fun findAllByRecruitmentId(recruitmentId: Long): List<ApplicantResponse> {
+        val applicationForms = applicationFormService.findAllByRecruitmentId(recruitmentId).map {
+            it.applicantId to it
+        }.toMap()
```

Comment:
```suggestion
        val applicationForms = applicationFormService.findAllByRecruitmentId(recruitmentId)
        .associate { it.applicantId to it }
```

코틀린 공부는 끝이 없네요. value값이 자기 자신이면 `.associaeBy {it.applicantId}`처럼 사용해도 되고요.